coding-concepts.js

```diff
@@ -7,48 +7,73 @@
 
 // --------------------INSTRUCTOR EXAMPLE: What will this log?
 const colors = ["tangerine", "magenta", "lilac", "daffodil"]
-// console.log(colors.push("indigo"))
+console.log(colors.push("indigo"))
 
-// a) Your answer:
+// a) Yourlsls answer: 
 // b) Verify and explain:
 
 // --------------------1) What will this log?
 
 const cohort = "LEARN 2023"
-// console.log(cohort.length)
+console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 10
+// b) Verify and explain:.length is modifying the reurn to output the length of the string defined by the functiong: const cohort
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
-// console.log(greeting[4])
+console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: o
+// b) Verify and explain: [4] tells the console log which index to display value
 
 // --------------------3) What will this log?
 
 const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
-// console.log(languages[index])
+console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: Ruby
+// b) Verify and explain: index being set to 1 indicates the second index in zero indexing.
 
 // --------------------4) What will this log?
 
 const weekendDays = ["saturday", "sunday"]
-// console.log(weekendDays.toUpperCase())
-
-// a) Your answer:
-// b) Verify and explain:
+console.log(weekendDays).toUpperCase
+
+// a) Your answer: SATURDAY, SUNDAY
+// b) Verify and explain: toUpperCase outputs the string in capital letters.
+ 
+// output error
+// removed () because looked weird
```

Comment:
Do not remove parentheses because they look weird. JavaScript syntax is very specific, so removing those parentheses will give you an unexpected output.